dapps/marketplace/src/hoc/withEnrolmentStatus.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react'
+import { Query } from 'react-apollo'
+import QueryError from 'components/QueryError'
+import get from 'lodash/get'
+
+import enrollmentStatusQuery from 'queries/EnrollmentStatus'
+import withWallet from './withWallet'
+
+function withEnrolmentStatus(
+  WrappedComponent,
+  { fetchPolicy = 'network-only', suppressErrors } = {}
+) {
+  let refetchStatus
+
+  const WithEnrolmentStatus = props => {
+    useEffect(() => {
```

Comment:
since props.wallet is part of the Query component's variables, wouldn't the refetch happen automatically?

Reply:
It should have updated when the variables changed. But for some reasons, I didn't see a refetch happening on my local build. I had posted about this on Discord too. If it works fine on Staging/Prod, I can revert this.

Reply:
It seems to be working on dev, staging and production. Must be something related to my local build. Have reverted this change.